src/inverted-index.js

```diff
@@ -6,99 +6,110 @@
  * @class
  */
 class InvertedIndexClass {
-    /**
-     * Class Instantiation.
-     */
+  /**
+   * Class Instantiation.
+   */
   constructor() {
     this.index = {};
     this.books = {};
     this.parsedBooks = {};
   }
-    
-    /**
-     * create Index
-     * 
-     * creates an index of words
-     * 
-     * @param{string}  filename accepts a string name for a file;
-     * @param{object} books is a json object for which an index is to be created
-     * @returns{object};
-     */
+
+  /**
+   * create Index
+   * 
+   * creates an index of words
+   * 
+   * @param{string}  filename accepts a string name for a file;
+   * @param{object} books is a json object for which an index is to be created
+   * @returns{object};
+   */
   createIndex(filename, books) {
     this.index[filename] = {};
-    this.books = books;
-
-    for (let i = 0; i < books.length; i += 1) {
-      const textIndex = books[i].text.toLowerCase().match(/[\w']+/g);
-      for (let j = 0; j < textIndex.length; j += 1) {
-        if (this.index[filename][textIndex[j]] === undefined) {
-          this.index[filename][textIndex[j]] = {};
-          this.index[filename][textIndex[j]][i] = true;
+    this.books = (books);
+    let bookIndex;
+    books.forEach((singleBook, inex) => {
```

Comment:
### Critical

Please use reasonable variable names, `inex` does not make sense.